client.lua

```diff
@@ -1,9 +1,5 @@
-local Driving = false;
 local Underwater = false;
 local enableCruise = false;
-local Keys = {
```

Comment:
The keys are used for the key mappings on lines 57 & 58.

Reply:
Ah i must have missed that, will make add the keys table back in